azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/arm/implementation/AzureConfigurableImpl.java

```diff
@@ -101,11 +101,15 @@ public T withProxyAuthenticator(Authenticator proxyAuthenticator) {
     }
 
     protected RestClient buildRestClient(AzureTokenCredentials credentials, AzureEnvironment.Endpoint endpoint) {
-        return restClientBuilder
+        RestClient client =  restClientBuilder
                 .withBaseUrl(credentials.environment(), endpoint)
                 .withCredentials(credentials)
                 .withInterceptor(new ProviderRegistrationInterceptor(credentials))
                 .build();
+        if (client.httpClient().proxy() != null) {
+            credentials.withProxy(client.httpClient().proxy());
```

Comment:
This is a known issue tracked via https://github.com/AzureAD/azure-activedirectory-library-for-java/blob/dev/src/main/java/com/microsoft/aad/adal4j/AuthenticationContext.java.